charts/datahub/subcharts/acryl-datahub-actions/templates/deployment.yaml

```diff
@@ -35,6 +35,12 @@ spec:
             defaultMode: 0444
             secretName: {{ .name }}
       {{- end }}
+      {{- with .Values.ingestionSslSecrets }}
+        - name: ingestion-ssl-secrets
```

Comment:
Is there a reason we need to mark this as "ssl-secrets"? 

Can we generalize a bit more to call it "ingestion-secret-files" or something similar. Since it may not _always_ be for SSL?

Reply:
Sure.